### PR TITLE
I added a disk page manager, B+Tree Node and a B+Tree Iterator

### PR DESCRIPTION
**Pull request by Skeletus**
The disk page manager manages binary files and divides them into pages, it is expected to implement a feature that allows calculating the b+tree order according to the size of the page

### CODE REVIEW

File: .gitignore
Comment (AngelloFD) [inline .gitignore:1]: I am using VSCode, these files wouldn't even be generated to begin with.

File: BTreeAdventure/BTreeAdventure.sln
Comment (AngelloFD) [inline BTreeAdventure/BTreeAdventure.sln:1]: VSCode doesn't generate a .sln file

File: BTreeAdventure/BTreeAdventure/BTreeAdventure.vcxproj
Comment (AngelloFD) [inline BTreeAdventure/BTreeAdventure/BTreeAdventure.vcxproj:1]: VSCode doesn't generate a .vcxproj file

File: BTreeAdventure/BTreeAdventure/BTreeAdventure.vcxproj.filters
Comment (AngelloFD) [inline BTreeAdventure/BTreeAdventure/BTreeAdventure.vcxproj.filters:1]: VSCode doesn't generate a .vcxproj.filters file

**Review by AngelloFD — COMMENTED** [review]
I'm not going to read from binary files and I am not using MVSCode as my IDE

File: BTreeAdventure/BTreeAdventure/BTreeAdventure.vcxproj.filters
Comment (Skeletus) [reply BTreeAdventure/BTreeAdventure/BTreeAdventure.vcxproj.filters:1]: sorry :c I'm like using VS 2022

**Pull request closed without merging**